gravitee-apim-rest-api/gravitee-apim-rest-api-portal/gravitee-apim-rest-api-portal-rest/src/main/java/io/gravitee/rest/api/portal/rest/resource/AbstractResource.java

```diff
@@ -182,6 +182,10 @@ protected void checkImageFormat(final String encodedPicture) {
             if (!mediaType.startsWith("image/")) {
                 throw new UploadUnauthorized("Image file format unauthorized " + mediaType);
             }
+
+            if (encodedPicture.contains("image/svg+xml")) {
```

Comment:
I'm wondering we should make that rule softer like
```suggestion
            if (encodedPicture.contains("svg")) {
```

🤔

Reply:
I think it could leave to false positive since the encoded string could randomly contains "svg"

Reply:
Indeed, maybe more

```suggestions
            if (encodedPicture.startWith("image/svg")) {
```